index.html

```diff
@@ -418,6 +419,163 @@ <h2>Transferable MediaStreamTrack</h2>
       <p>The underlying source is supposed to be kept alive between the transfer and transfer-receiving steps, or as long as the data holder is alive.
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
+    <h3>MediaStreamTrack frame stats</h3>
+    <div>
+      <p>
+        On any track that supports {{MediaTrackSettings/frameRate}} as a
```

Comment:
This seems to indicate that peer connection tracks would have this counter.
Does this make sense? It seems we mostly care about cameras and screen share.
Can we find wording to do this?

Reply:
Done. I reworded this as "is sourced from getUserMedia() or getDisplayMedia()"

Reply:
OK, this seems good enough as a first iteration.
In the future, we might want to add a [hasFrameStats] slot to the track's source.
If the slot is not there or is false, we would reject getFrameStats().
mediacapture-main and mediacapture-screenshare specs would then set this slot to true.

Reply:
Good suggestion for the future! Then we can let each individual spec decide.